src/main/java/ch/wisv/areafiftylan/service/OrderServiceImpl.java

```diff
@@ -36,10 +37,11 @@
 
     @Autowired
     public OrderServiceImpl(OrderRepository orderRepository, UserService userService, TicketRepository ticketRepository,
-                            PaymentService paymentService, ExpiredOrderRepository expiredOrderRepository) {
+                            TicketService ticketService, PaymentService paymentService, ExpiredOrderRepository expiredOrderRepository) {
```

Comment:
It's weird that we use both the TicketRepository and the TicketService here. That's not part of this PR, but a reminder that I should fix this.